### PR TITLE
TBS - OKTA-458445 Adding release notes for 2022.01.1

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adds release notes for 2022.01.1
- **Is this PR related to a Monolith release?** Yes, 2022.01.1

### Resolves:

* [OKTA-458445](https://oktainc.atlassian.net/browse/OKTA-458445)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 1]
```diff
@@ -6,6 +6,20 @@ title: Okta Identity Engine API Products release notes 2022
...
 
+### Weekly release 2022.01.1
+
+| Change                                                                                              | Expected in Preview Orgs |
+|-----------------------------------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.01.1](#bugs-fixed-in-2022-01-1)         | January 13, 2022           |
+
+#### Bugs fixed in 2022.01.1
+
+* If the [Create a Scope](/docs/reference/api/authorization-servers/#create-a-scope) endpoint received multiple requests at or near the same time, duplicate Scopes could be created. (OKTA-442533)
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:17]: duplicate Scopes << should "Scopes" be initial uppercase here? I've noticed that scopes is usually lowercase in the dev docs, or is this special?

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 2]
```diff
@@ -3,6 +3,20 @@ title: Okta API Products release notes 2022
...
 
+### Weekly release 2022.01.1
+
+| Change                                                                                              | Expected in Preview Orgs |
+|-----------------------------------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.01.1](#bugs-fixed-in-2022-01-1)         | January 13, 2022           |
+
+#### Bugs fixed in 2022.01.1
+
+* If the [Create a Scope](/docs/reference/api/authorization-servers/#create-a-scope) endpoint received multiple requests at or near the same time, duplicate Scopes could be created. (OKTA-442533)
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:14]: Same comment as above.

**Review by christinechoy-okta — APPROVED** [review]
Just a query, otherwise LGTM!

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md (hunk 1 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:17]: Good question. It's capitalized in the api docs to which I link. That's what prompted me to capitalize it here. I see that we don't capitalize it in the guides. I think I'll capitalize the link text to align with the reference doc but leave the other use of scope in lowercase.

**Pull request merged**